src/org/opendatakit/briefcase/ui/export/AggregateServerConnectionConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package org.opendatakit.briefcase.ui.export;
```

Comment:
Wrong package... Should be `org.opendatakit.briefcase.ui.reused`

Reply:
:+1: